### DISCUSSION TITLE
Pokeball not using specified on uncaught

### DISCUSSION THREAD

**Discussion by Tj-Burger** [Farigh/pokeclicker-automation#302] [Q&A] [answered]
Not sure what I'm doing wrong or if it's a bug, I have the focus on set to off. I am trying to catch roamers in an area so I have my "new" pokeball set to masterball and the "automation" in the window set to masterball yet when I check logs it shows encountered defeated and got away. I'm assuming it is using some other setting when encountering uncaught pokemon but I can't find anything that's enabled that would prevent this from working. Is there a setting I am overlooking or something to check?

**Comment by Farigh** [comment, marked as answer]
Hi,

I'm not sure to understand what you're trying to do.

If the 'Focus on' is off, it will disable the 'Automation' catch setting. If you want to catch Pokemon without using the automation features, you should use the game default settings, or create your own.
